analytics/terraform/spark-k8s-operator/versions.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 1.0.0"
+  required_version = ">= 1.3.2"
```

Comment:
Do we need to do this for others? If so, we can probably do the others in another PR.

Reply:
good call, I need to check the other blueprints and get a followup together.

I think the `Min TF pre-commit` will break in the checks so we should be able to catch them if I miss any.

Reply:
Awesome. I'd say it's good to merge this. Thanks so much for the effort!